### PR TITLE
chore(blog): Gatsby CLI/Error blogpost

### PR DESCRIPTION
**Pull request by shannonbux**
@KyleAMathews @lindaleebumblebee

### CODE REVIEW

File: docs/blog/2019-05-20-cli-error-redesign/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Lions and Tigers and Errors, Oh My! Redesigning the Gatsby CLI
```
Comment (marcysutton) [inline docs/blog/2019-05-20-cli-error-redesign/index.md]: Ha ha, love the title! 🐅 🚫

File: docs/blog/2019-05-20-cli-error-redesign/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - ux
+---
+
+## Why did we do this study?
+
+Our command-line interface (CLI) is OK but not great. It will be useful to generate a set of design principles to base the CLI around — but not change what's working well (which is a lot of it). So let’s do it!
+
+The CLI is the main entry point and interface people have (currently) for Gatsby. Everyone sees it! So it has a huge influence on people’s experience of Gatsby.
+
+We’re going to create a CLI style guide and then implement that style guide, which includes redesigning error messages, introducing `gatsby --quiet` as a default mode and make more extensive use of `gatsby --verbose` flag. To make the CLI easier to design and improve, we’ll implement Ink. To optimize the CLI design, we’ll use the data being collected through our telemetry setup, which collects data about CLI usage (opt out is possible).
```
Comment (marcysutton) [inline docs/blog/2019-05-20-cli-error-redesign/index.md]:
```suggestion
We’re going to create a CLI style guide and then implement that style guide, which includes redesigning error messages, introducing `gatsby --quiet` as a default mode and make more extensive use of the `gatsby --verbose` flag. To make the CLI easier to design and improve, we’ll implement [Ink](https://github.com/vadimdemedes/ink). To optimize the CLI design, we’ll use the data being collected through our telemetry setup, which collects data about CLI usage (opt out is possible).
```

File: docs/blog/2019-05-20-cli-error-redesign/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## What did we learn?
+
+4 usability tests with awesome Gatsby users to learn how they do tasks using the Gatsby CLI and how they use error messages to debug their projects. Thanks [Simon Koelewijn](https://github.com/smnk), [Gene Smith](https://twitter.com/gene_r_smith), [Jonathan Prozzi](https://github.com/jonathanprozzi), and [Benjamin Lannon](https://github.com/lannonbr) for teaching us how you use the CLI, and thanks to [@sidharthachatterjee](https://github.com/sidharthachatterjee), [@wardpeet](https://github.com/wardpeet), [@pieh](https://github.com/pieh), [@m-allanson](https://github.com/m-allanson), [@gillkyle](https://github.com/gillkyle) for copiloting the interviews with me and helping turn what we learned into action.
```
Comment (marcysutton) [inline docs/blog/2019-05-20-cli-error-redesign/index.md]:
```suggestion
We conducted 4 usability tests with awesome Gatsby users to learn how they do tasks using the Gatsby CLI and how they use error messages to debug their projects. Thanks [Simon Koelewijn](https://github.com/smnk), [Gene Smith](https://twitter.com/gene_r_smith), [Jonathan Prozzi](https://github.com/jonathanprozzi), and [Benjamin Lannon](https://github.com/lannonbr) for teaching us how you use the CLI, and thanks to [@sidharthachatterjee](https://github.com/sidharthachatterjee), [@wardpeet](https://github.com/wardpeet), [@pieh](https://github.com/pieh), [@m-allanson](https://github.com/m-allanson), [@gillkyle](https://github.com/gillkyle) for copiloting the interviews with me and helping turn what we learned into action.
```

**Comment by shannonbux** [comment]
@lindaleebumblebee I think this is ready to merge. Any comments?

**Comment by lindaleebumblebee** [comment]
The title is so great :). Looks good to me. Feel free to post anytime this week.

**Pull request merged**